creatorkit/Toolbox.md

```diff
@@ -0,0 +1,135 @@
+![Creator Logo](../creatorlogo.png)
+
+# Toolbox
+
+The toolbox is a reference page for you to use to set up your kit. Follow the steps listed in the project pages and refer back to this page for more detail on individual steps.
+
+## Hardware
+
+* Creator Kit
+* Ubuntu 14.04 64-bit PC/Laptop
+
+## Powering Ci40 + Clicker boards
+
+There are 2 ways to power your Ci40 board.
+
+* Using a power supply
+* Using a microUSB cable connected to the port next to the power supply port
```

Comment:
it's microUSB. First letter is only capitalised because it's the start of a sentence.